tools/generate-intrinsics-specs.sh

```diff
@@ -53,4 +53,6 @@ generate_rst_specs ./tools/intrinsic_db/mve.csv \
 
 # Check changes
 check_changes ./tmp/advsimd.new.rst ./neon_intrinsics/advsimd.rst
-#check_changes ./tmp/mve.new.rst ./mve_intrinsics/mve.rst
+check_changes ./tmp/mve.new.md ./mve_intrinsics/mve.md
+markdown-link-check --verbose --config package-lock.json mve_intrinsics/mve.md
```

Comment:
Maybe add a comment like `# Check links and hyperlinks` to separate from the `Check changes` step?